emp-wrapper/src/main/java/com/ballerina/sf/BayeuxParameters.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2016, salesforce.com, inc. All rights reserved. Licensed under the BSD 3-Clause license. For full
+ * license text, see LICENSE.TXT file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package com.ballerina.sf;
```

Comment:
Shall we change the package to `org.ballerinalang.sf`? We use the same convention in other modules